### PR TITLE
REYFM buttons

### PR DESCRIPTION
**Pull request by Bas950**
image

### CODE REVIEW

File: websites/R/REYFM/presence.ts [hunk 1]
```diff
@@ -183,6 +196,8 @@ presence.on("UpdateData", async () => {
       presenceData.state = format2
         .replace("%title%", track)
         .replace("%artist%", artist);
+
+      presenceData.buttons = [{ label: "Listen along!", url: document.URL }];
```
Comment (ririxidev) [inline websites/R/REYFM/presence.ts:200]: You sure it will not redirect to main page in any way?

File: websites/R/REYFM/presence.ts [hunk 2]
```diff
@@ -213,6 +228,8 @@ presence.on("UpdateData", async () => {
 
     showFormat3 = true;
 
+    presenceData.buttons = [{ label: "Listen along!", url: document.URL }];
```
Comment (ririxidev) [inline websites/R/REYFM/presence.ts:231]: You sure it will not redirect to main page in any way?

**Review by ririxidev — CHANGES_REQUESTED** [review]
.

File: websites/R/REYFM/presence.ts (hunk 1 above)
Comment (Bas950) [reply websites/R/REYFM/presence.ts]: Fixed.

File: websites/R/REYFM/presence.ts (hunk 2 above)
Comment (Bas950) [reply websites/R/REYFM/presence.ts:232]: This one is always in a /gaming or a /channel thingy

**Review by ririxidev — APPROVED** [review]
ok

**Review by StrikerFRFX — APPROVED** [review]
lgtm 😄

**Pull request merged**